guillotina/contrib/dbusers/serializers.py

```diff
@@ -39,6 +40,8 @@ def __init__(self, context, request):
             self.context.email = data["email"]
         if "fullname" in data:
             self.context.name = data["fullname"]
+        if "user_groups" in data:
+            self.context.user_groups = data["user_groups"]
```

Comment:
As a future improvement, maybe user groups, roles, permissions, and so on, would be nice to have them defined in the schema as a `PatchField(List)`, so we can have partial updates aswell?

Reply:
Mmm.. not now..  this is a braking change that would brake guillotina react..

Reply:
PatchField works as regular field as well.